src/build/env.py

```diff
@@ -67,7 +67,9 @@ def __enter__(self):  # type: () -> IsolatedEnvironment
             if path in sys_path:
                 sys_path.remove(path)
 
-        self._replace_env('PATH', self._get_env_path('scripts'))
+        self._replace_env('PATH', os.pathsep.join(
+            filter(None, [os.environ.get('PATH', None), self._get_env_path('scripts')]))
```

Comment:
Two things:

1. Would it make sense for `PATH` to come second here, so that anything installed in the build environment would take precedence over globally-installed tools?
2. `filter(None, ...)` will also filter out things like `''`, and apparently `PATH=''` is a valid setting (it makes it so that only the CWD is on the PATH). You can change the `None` to `lambda x: x is None`, or possibly it's cleaner to break it across multiple lines (can `self._get_env_path` ever return `None`?).

Reply:
1) Oh yes, that was my intent, sorry.

2) Right, I forgot `filter` had this special handling for `None`. It is helpful most of the time, but not here.

Reply:
This change request here by @pganssle still needs addressed.